frontend/components/apps/NewAppDialog.tsx

```diff
@@ -32,6 +32,7 @@ import {
   getUserKxPrivateKey,
   getUserKxPublicKey,
 } from '@/utils/crypto'
+import { MAX_INPUT_STRING_LENGTH, MAX_STRING_LENGTH } from '@/constants'
```

Comment:
```suggestion
import { MAX_INPUT_STRING_LENGTH } from '@/constants'
```

I think only one import is required here